### DISCUSSION TITLE
prisma/codemods doesn't transform findUnique

### DISCUSSION THREAD

**Discussion by pasekit** [prisma/prisma#5247] [Q&A] [answered]
I just did an update from version 2.10.1 to version 2.15.0. Furthermore I ran "npx @prisma/codemods update-2.12 ./". Then times only the single findUnique transform. However always the same result comes out:

    PS C:\App\Backend> npx @prisma/codemods findUnique ./ --print
    Executing command: jscodeshift --print --verbose=0 --ignore-config=.gitignore --ignore-pattern=**/node_modules/** -- 
    extensions=tsx,ts,js,tsx --parser=tsx --transform 
    C:\App\Backend\node_modules\@prisma\codemods\transforms\findUnique.js ./
    No files selected, nothing to do.
    All done.
    Results:
    0 errors
    0 unmodified
    0 skipped
    0 ok
    Time elapsed: 0.035seconds

Why does codemods not recognize the files in my project?

**Comment by 2color** [comment, marked as answer]
Hey @pasekit 👋🏼,

This looks like a bug. 

Could you please open an issue in the [`prisma/codemods`](https://github.com/prisma/codemods/issues/new/choose) repository?

Would be great if you could also share your folder structure with [tree](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/tree).

**Reply by pasekit** [reply]
i was afraid of that. ok, then i will create the issue. thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
